.github/workflows/neko.yml

```diff
@@ -0,0 +1,16 @@
+name: Qiskit Neko Integration Tests
```

Comment:
I have node idea whether anything will be done with this. I can imagine it being removed for apps.